building/trusted_applications.rst

```diff
@@ -373,35 +373,39 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the so-called `in-tree` TAs, which come
+with the OPTEE OS (e.g. the ``pkcs11`` TA) and any external TA coming from the user. Offline signing
```

Comment:
s/OPTEE/OP-TEE/ at all the places in this PR.

Reply:
@jenswi-linaro You mean literally to replace "OPTEE OS" with "s/OPTEE/OP-TEE/"? As in copy-past?

Reply:
No, only the string "OPTEE" with "OP-TEE".

Reply:
Done!